docs/scripts/script.min.js

```diff
@@ -21509,7 +21509,8 @@ class PokemonFactory {
             return new BattlePokemon('MissingNo.', 0, PokemonType.None, PokemonType.None, 0, 0, 0, 0, new Amount(0, GameConstants.Currency.money), false, 0);
         }
         let name;
-        if (PokemonFactory.roamingEncounter(route, region)) {
+        const roamer = PokemonFactory.roamingEncounter(route, region)
+        if (roamer) {
```

Comment:
Don't submit anything in the docs-folder. The buildscript will be run for each release.

Reply:
I'll revert this